src/Auth0.ManagementApi/HttpClientManagementConnection.cs

```diff
@@ -46,20 +44,7 @@ public HttpClientManagementConnection(HttpMessageHandler handler)
         }
 
         /// <inheritdoc />
-        public void SetDefaultHeaders(IDictionary<string, string> headers)
-        {
-            if (ownHttpClient)
-            {
-                ApplyHeaders(httpClient.DefaultRequestHeaders, headers);
-            }
-            else
-            {
-                defaultHeaders = headers;
-            }
-        }
-
-        /// <inheritdoc />
-        public Task<T> GetAsync<T>(Uri uri, IDictionary<string, string> headers = null, JsonConverter[] converters = null)
+        public Task<T> GetAsync<T>(Uri uri, IDictionary<string, string> headers, JsonConverter[] converters = null)
```

Comment:
Curious why we're making this parameter required now? I guess we'll always have, at the very least, an auth header and telemetry ...

Reply:
Two-fold.

1. You should never be using this without default headers
2. I wanted to see if I missed passing any parameters in when I added DefaultHeader to the callers.